### ISSUE TITLE
Detection List 

### ISSUE THREAD

**Issue by E3V3A** [CellularPrivacy/Android-IMSI-Catcher-Detector#230]
Here is a list of the various detection items (_flags_) that we wish to implement. It's for developer reference only, and the details surrounding each item should (eventually) be found in our technical developer Wiki documents. So please don't ask questions **here** about how they work. We have a chatroom for that, which you can ask to be added to, if you think you have something important to share or contribute with. 

| DB_id | GitHub | ShortName | API | Status | NM |
| :-- | :-- | :-- | :-- | :-: | :-: |
| 1 | #253 | DBe Consistency Check | * | 1 |  |
| 2 | #91 | LAC/CID Consistency | * | 2 |  |
| 3 | #73 | T3212 Registrations | - |  |  |
| 4 | #93 | TMSI changes | - |  |  |
| 5 | (#96) | Ciphering Consistency | - |  |  |
| 6 |  | A5/x Downgrade | - |  |  |
| 7 | #100 | Neighbors Consistency | / |  |  |
| 7a | #264 #208 | Check BTS for the NC List | / | 1 |  |
| 8 |  | RX signal Quality | * |  | * |
| 9 | #97 | RX Signal Strength | * | 1 | * |
| 10 |  | TX Signal Power | - |  | * |
| 11 | #176 | MNO RAT Consistency | * |  |  |
| 12 | #69 | Silent SMS/WAP Push | / | 1 |  |
| 13 | #151 | Silent App Install | - | 1 |  |
| 14 |  | Silent Calls | ? |  |  |
| 15 |  | MDM Manipulation | - |  |  |
| 16 |  | TA inconsistency | / |  | * |
| 17 | #82 | LTE/3G downgrade | * |  |  |
| 18 |  | Out-of-band usage | - |  |  |
| 19 |  | MAR out-of-bounds | * |  | * |
| 20 | #6 | FemtoCell Detection | - | 1 | * |
| 21 | #221 | QC BP Signal Jamming | - |  |  |
| 22 |  | LTE RSRQ/RSRP analysis | / |  |  |
- NM = Non Moving condition (Device must not move for this detection to work.)
- `[*,/,-]` = Is present in AOS API?  `[Yes,  Partially,  No ]`

Implementation Status:

> 0 / empty = not implemented
> 1 = work in progress
> 2 = implemented and complete
> 3 = deprecated

---

**Q:** _How do we intend to use this?_
**A:** Each item on the detection list will have 3 associated values that will be added up individually to determine the final [threat level](https://github.com/SecUpwN/Android-IMSI-Catcher-Detector/wiki/Status-Icons) (detection status color + _icon_.) The 3 values are:
- **p1**: A "color code"; Used to give a rough measure of variable precedence.
- **p2**: A "Variable Interception Priority"; To what extent the variable is used to for tracking your network connections.
- **p3**: A "Variable Localization Priority": To what extent the variable is used to localize the victim.
##

**Comment by SecUpwN** [comment]
@E3V3A, I like that you keep track about what our app is aiming to do, but do you allow me to move this Issue into our [Development Status](https://github.com/SecUpwN/Android-IMSI-Catcher-Detector/wiki/Development-Status)? @DJaeger and I agree that Issues shall stay closeable at all times.

**Comment by E3V3A** [comment]
no, why would you wanna do that? This issue is referenced all over the place, so it's very bad business to move stuff like that.

**Comment by SecUpwN** [comment]
>no, why would you wanna do that?

To prevent misuse of the Issue system.

**Comment by E3V3A** [comment]
I agree, but I don't think this issue should be removed or changed at this point. Feel free to copy it over to Dev Stat so we can see how it looks. It's a huge advantage to have all single issue entries point pack to this one for quick overview.

**Comment by SecUpwN** [comment]
>I agree, but I don't think this issue should be removed or changed at this point.

Ok, fair. I will keep it as it is for now, although I guess moving will just get harder later.

>Feel free to copy it over to Dev Stat so we can see how it looks.

We currently point to this Issue from DevStat, also from our `README`. It will look good, trust me. ;-)

**Comment by DJaeger** [comment]
I think in the wiki we can not see an automaticly generated badge if the issue is open, right?
So leave it open.
If we closed all referenced issues we can alo close this.
The other issues should anyway be closed before ```BETA```, so this one will not stop us.

**Comment by SecUpwN** [comment]
>I think in the wiki we can not see an automatically generated badge if the issue is open, right?

Not yet. But I took the lead to file a GitHub feature request so that it may be available in the future.

>If we closed all referenced issues we can also close this.

Ok, fair. Looks like this will stay open forever then. Let's continue work. ;-)
